modules/skybrowser/skybrowsermodule.cpp

```diff
@@ -0,0 +1,692 @@
+                "",
+                "Starts the setup process of the sky browers. This function calls "
+                "the lua function 'sendOutIdsToBrowsers' in all nodes in the cluster."
+            },
+            {
+                "translateScreenSpaceRenderable",
+                &skybrowser::luascriptfunctions::translateScreenSpaceRenderable,
+                "string, float, float, float, float",
+                "Takes an identifier to a sky browser or sky target and the [x, y] "
+                "starting position and the [x, y] translation vector."
+             },
+        };
+
+        return res;
+    }
+
+SkyBrowserModule::SkyBrowserModule()
+    : OpenSpaceModule(SkyBrowserModule::Name)
+    , _allowMouseInteraction(AllowInteractionInfo, true)
+    , _allowCameraRotation(AllowRotationInfo, true)
+{
+    addProperty(_allowMouseInteraction);
+    addProperty(_allowCameraRotation);
+
+    // Set callback functions
+    global::callback::mouseButton->emplace_back(
+        [&](MouseButton button, MouseAction action, KeyModifier modifier) -> bool {
+            if (!_isCameraInSolarSystem || !_allowMouseInteraction) {
+                return false;
+            }
+
+            if (action == MouseAction::Press) {      
+                _isCameraRotating = false;
+                if (_mouseOnPair) {
+                    handleMouseClick(button);
+                    return true;
+                }
+                return false; 
+            }
+            else if (_interactionMode != MouseInteraction::Hover && 
+                     action == MouseAction::Release) {
+             
+                _interactionMode = MouseInteraction::Hover;
+                return true;
+            }
+            else {
+                return false;
+            }
+        }
+    );
+
+    if (global::windowDelegate->isMaster()) {
+        global::callback::mousePosition->emplace_back(
+            [&](double x, double y) {
+                if (!_isCameraInSolarSystem || !_allowMouseInteraction) {
+                    return false;
+                }
+
+                glm::vec2 pixel{ static_cast<float>(x), static_cast<float>(y) };
+                _mousePosition = skybrowser::pixelToScreenSpace2d(pixel);
+                glm::vec2 translation = _mousePosition - _startMousePosition;
+
+                switch (_interactionMode) {
+                    case MouseInteraction::Hover:
+                        setSelectedObject();
+                        break;
+                    case MouseInteraction::Drag:
+                        _mouseOnPair->translateSelected(_startDragPosition, translation);
+                        break;
+                    case MouseInteraction::FineTune:
+                        _mouseOnPair->fineTuneTarget(_startDragPosition, translation);
+                        break;
+                    default:
+                        setSelectedObject();
+                        break;
+                    }
```

Comment:
Nitpicky, but this closing bracket should align with the start of the switch statements. So, reduce indentation by 4 spaces